specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerGroup.json

```diff
@@ -0,0 +1,505 @@
+            "schema": {
+              "$ref": "#/definitions/NetworkGroupListResult"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkGroupsList": {
+            "$ref": "./examples/NetworkManagerGroupList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/networkGroups/{networkGroupName}/effectiveVirtualNetworks": {
+      "get": {
+        "tags": [
+          "NetworkGroups"
+        ],
+        "operationId": "EffectiveVirtualNetworks_ListByNetworkGroup",
+        "description": "Get the specified effective VirtualNetworks list by network group.",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NetworkGroupNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ListPageSizeParameter"
+          },
+          {
+            "$ref": "#/parameters/ListSkipTokenParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the network group.",
+            "schema": {
+              "$ref": "#/definitions/EffectiveVirtualNetworksListResult"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkGroupsList": {
+            "$ref": "./examples/NetworkManagerEffectiveVirtualNetworksList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkGroup": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkGroupProperties",
+          "description": "The Network Group properties"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
```

Comment:
Is NetworkGroup a proxy or tracked resource? I see tags defined for this resource. Any reason for referring ProxyResource properties here?

Reply:
Only NetworkManager is traced resource which is top level. All other resources are its children resources which should be proxy. But we want all resources (include networkgroup) can contain tags. Please let us know if it is impossible, and we can discuss it. Thanks!

Reply:
For proxy only resources, location and tags are not applicable in the request body. Refer https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope
Any reason why the nested resources can't be exposed as tracked resources?

Reply:
We want the chilren resources inherite location from their parent resources.

Reply:
As discussed in the meeting, please remove tags from proxy resources